conans/test/integration/toolchains/cmake/test_cmaketoolchain.py

```diff
@@ -7,6 +7,7 @@
 from mock import mock
 
 from conan.tools.cmake.presets import load_cmake_presets
+from conan.tools.files import load
```

Comment:
@franramirez688  Do not use ``from conan.tools.files``, use ``from conans.utils.files`` for testing.
The idea is that ``from conan.tools`` is imported estrictly from recipes, not used by testing code. The ugly ``None`` first parameter will be avoided.